components/card/index.css

```diff
@@ -10,40 +10,168 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+  --spectrum-card-corner-radius: var(--spectrum-corner-radius-100);
+  --spectrum-card-border-color: var(--spectrum-gray-200);
+  --spectrum-card-border-color-hover: var(--spectrum-gray-300);
+  --spectrum-card-border-color-selected: var(--spectrum-blue-700);
+
+  /* Typography */
+  --spectrum-card-title-font-family: var(--spectrum-sans-font-family-stack); 
+  --spectrum-card-title-font-size: var(--spectrum-heading-size-xxs);
+  --spectrum-card-title-font-weight: var(--spectrum-heading-sans-serif-font-weight);
+  --spectrum-card-title-font-style: var(--spectrum-heading-sans-serif-font-style);
+  --spectrum-card-title-line-height: var(--spectrum-heading-line-height);
+  --spectrum-card-title-color: var(--spectrum-heading-color);
+
+  --spectrum-card-body-font-family: var(--spectrum-sans-font-family-stack);
+  --spectrum-card-body-font-size: var(--spectrum-body-size-s);
+  --spectrum-card-body-font-weight: var(--spectrum-body-sans-serif-font-weight);
+  --spectrum-card-body-font-style: var(--spectrum-body-sans-serif-font-style);
+  --spectrum-card-body-line-height: var(--spectrum-body-line-height);
+  --spectrum-card-body-font-color: var(--spectrum-body-color);
+
+  /* Quick Actions */
+  --spectrum-card-actions-spacing: var(--spectrum-spacing-300);
+  --spectrum-card-actions-size: var(--spectrum-card-selection-background-size);
+  --spectrum-card-actions-border-radius: var(--spectrum-corner-radius-100);
+  /* TODO update to --spectrum-card-selection-background-color token once an RGB stripped value is available */
+  --spectrum-card-actions-background-color-rgb: var(--spectrum-gray-100-rgb);
+  --spectrum-card-actions-background-color-opacity: var(--spectrum-card-selection-background-color-opacity);
+  --spectrum-card-actions-drop-shadow-color: var(--spectrum-drop-shadow-color);
+  --spectrum-card-actions-drop-shadow-x: var(--spectrum-drop-shadow-x);
+  --spectrum-card-actions-drop-shadow-y: var(--spectrum-drop-shadow-y);
+  --spectrum-card-actions-drop-shadow-blur: var(--spectrum-drop-shadow-blur);
+
+  /* Focus */
+  --spectrum-card-focus-indicator-color: var(--spectrum-focus-indicator-color);
+  --spectrum-card-focus-indicator-width: var(--spectrum-focus-indicator-thickness);
+
+  /* Selected */
+  --spectrum-card-selected-background-opacity: 0.1; /* table-selected-row-background-opacity does not exist in tokens yet */
+  /* TODO: These are placeholder until recursive RGB is available */
+  .spectrum--light &,
+  .spectrum--lightest & {
+    --spectrum-card-selected-background-color-rgb: var(--spectrum-blue-900-rgb);
+  }
+  .spectrum--dark & {
+    --spectrum-card-selected-background-color-rgb: var(--spectrum-blue-500-rgb);
+  }
+  .spectrum--darkest & {
+    --spectrum-card-selected-background-color-rgb: var(--spectrum-blue-600-rgb);
+  }
 
-.spectrum-Card--sizeS {
-  @remapvars {
-    find: /--spectrum-card(.*)-s-/;
-    filter: color;
-    replace: --spectrum-card$1-;
+  /* Quiet/Gallery */
+  &--quiet,
+  &--gallery {
```

Comment:
OMG so exciting to see this approach in action!

Reply:
@castastrophe question I just thought of. Should the variant variables be mods or not? 
`--spectrum-card-content-margin-top--quiet` vs `--mod-card-content-margin-top--quiet`? Since we are having them set mods normally, I'm wondering if these should also be mods to avoid confusion?